### DISCUSSION TITLE
Qual a melhor arquitetura?

### DISCUSSION THREAD

**Discussion by cs-wesleymarcolino** [concretesolutions/concrete_android_community#5] [Pergunte à comunidade]
Qual a melhor arquitetura?

**Comment by cs-wesleymarcolino** [comment]
Gosto do MVVM e vcs?

**Reply by cs-wesleymarcolino** [reply]
MVVM é o melhor mesmo, concordo plenamente.

**Reply by cs-wesleymarcolino** [reply]
Mas e o MVI hein?

![15cab6c6fb28ce1718b87e60040f69c6](https://user-images.githubusercontent.com/33731347/117014157-e88d2b80-acc6-11eb-97cc-b99288acf823.gif)

**Reply by cs-cristianocortez** [reply]
MVC raiz !

**Comment by CS-ConcreteAmorim** [comment]
Vou de MVP e vc ?
